docs/datamodel/constraints.rst

```diff
@@ -281,7 +281,7 @@ itself*. This is commonly used add constraints to :ref:`link properties
     type User {
       name: str;
       multi friends: User {
-        single strength: float64;
+        strength: float64;
```

Comment:
It didn’t seem like `single` was needed here. But please correct me if I am wrong about this.

Reply:
Agreed, don't see any need for `single` here.